From 0266c3d411fe01776f995ecee99de73ebbbc65ac Mon Sep 17 00:00:00 2001
From: Mathew McBride <matt@traverse.com.au>
Date: Mon, 16 Dec 2024 14:11:23 +1100
Subject: [PATCH] platform: raspberry-pi: remove RPI reference for default SSID
 naming

Our changes for RDK-B on generic Arm boards don't necessitate a
new platform/ folder yet, so use a simple patch in the mean time

Change-Id: Ibc6d1d8e373f1e648cd4534d986c96eeae77ca08
Signed-off-by: Mathew McBride <matt@traverse.com.au>
---
 platform/raspberry-pi/platform_pi.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/platform/raspberry-pi/platform_pi.c b/platform/raspberry-pi/platform_pi.c
index 071c818..e450a14 100644
--- a/platform/raspberry-pi/platform_pi.c
+++ b/platform/raspberry-pi/platform_pi.c
@@ -139,7 +139,7 @@ int platform_get_keypassphrase_default(char *password, int vap_index)
 int platform_get_ssid_default(char *ssid, int vap_index)
 {
     wifi_hal_dbg_print("%s:%d \n",__func__,__LINE__);   
-    sprintf(ssid,"RPI_RDKB-AP%d",vap_index);
+    sprintf(ssid,"RDKB-AP%d",vap_index);
     return 0;
 }
 
-- 
2.39.5

